### DISCUSSION TITLE
[Q&A] Agent Monitor automatic installation and startup

### DISCUSSION THREAD

**Discussion by kintaro1981** [glpi-project/glpi-agent#448] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

I saw that from glpi agent 1.5 is possibile to install GLPI Agent Monitor directly from the msi with AGENTMONITOR=1 (with EXECMODE=1).

My question is: MSI is going to set it to start automatically or we need to add it "manually" to the startup folder like said [here](https://glpi-agent.readthedocs.io/en/latest/installation/index.html#glpi-agent-monitor)?

**Comment by g-bougard** [comment]
Hi @kintaro1981 

no, in that case, you don't have to add it manually as the installer configure the system to start GLPI-Agent monitor for a user when he logs in.
And so yes, it starts automatically for each user.

I have to add a hint on the documentation to tell this is not the case if installed from glpi-agent MSI installer.

**Reply by g-bougard** [reply]
I added the hint, it should appear soon.

Thank you for your suggestive question.
